Updating wording on SKE page (#3567)
DFE-Digital/get-into-teaching-app: app/views/content/how-to-apply-for-teacher-training/subject-knowledge-enhancement.md

## How to do an SKE course

> If you think you need to improve your subject knowledge then talk to your training provider when you apply for teacher training in England. They’ll help you to decide if you'll benefit from a SKE course. 

Reviewer: ```suggestion
If you think you need to improve your subject knowledge then talk to your training provider when you apply for teacher training in England. They’ll help you to decide if you'll benefit from an SKE course. 
```
UR showed that candidates spell out 'SKE' when saying it (es-kay-ee) so we should be using 'an' when talking about it (pronouncing it 'ski' is just what we do internally)

> You can only complete a funded SKE course if your training provider makes it a condition of the offer to train with them.

Reviewer: does this mean you can complete an un-funded SKE course even if the provider doesn't make it a condition of their offer? worth clarifying

## How to do an SKE course

> If you think you need to improve your subject knowledge then talk to your training provider when you apply for teacher training in England. They’ll help you to decide if you'll benefit from a SKE course. You can only complete a funded SKE course if your training provider makes it a condition of the offer to train with them.

Reviewer 2: ```suggestion
If you think you need to improve your subject knowledge, talk to your training provider when you apply for teacher training in England. They’ll help you to decide if you'll benefit from a SKE course. You can only complete a funded SKE course if your training provider makes it a condition of the offer to train with them.
```

Reviewer 2: @philspray I don't know whether `when you apply for teacher training in England` is quite right - wouldn't it be before you apply? However, I appreciate this is what is says in production at the moment!

Worth clarifying with the SKE team

Reviewer 2: Would it be worth splitting `You can only complete......` onto a new paragraph? Feels like a lot of info to take in here

Reviewer 2: I'm not sure `help YOU to decide` is right - it is the provider who makes SKE a condition of offer, not you choosing whether to do it or not? Worth clarifying with the SKE team

> You can choose your SKE course provider, but your teacher training provider can help you decide on the right one for you.

Reviewer 2: Think this was probably better as `You'll` rather than have two `can` in the sentence?

### SKE subjects

> 

Reviewer 2: @philspray Presume you deliberately left out the `(8 weeks)` after religious education? I agree - this is irrelevant in the list of subjects

They're usually online, but some courses can be classroom based and some can be a mixture of both. 

> Course lengths usually range from 8 to 28 weeks and are 25 hours per week. Your provider will decide which course length you need based on your gap in knowledge. Different providers may offer different course lengths.

Reviewer 2: ```suggestion
Course lengths range from 8 to 28 weeks and are 25 hours per week. Your provider will decide which course length you need based on your gap in knowledge. Different providers may offer different course lengths.
```
SKE team suggested removing `usually` - makes sense if all courses are between 8 and 28 weeks?

You can do a part-time course and complete it over a longer period or complete it in a shorter time with compressed hours. Your SKE provider will be able to discuss your options with you.

## When to complete your course

You should complete your course before starting your teacher training. If you run out of time, your provider may allow you to continue your course at the same time as your teacher training.

## Get an SKE bursary

> If you train to teach in the 2022/23 academic year, you could be eligible for a bursary of £175 per week to complete your SKE course. If you do a part-time course and complete less than 25 hours a week, this will be paid in proportion to your course hours. If you complete your SKE course with compressed hours, this will affect your bursary amount. How and when your bursary is paid will vary by your training provider.

Reviewer 2: ```suggestion
If you train to teach in the 2023/24 academic year, you could be eligible for a bursary of £175 per week to complete your SKE course. If you do a part-time course and complete less than 25 hours a week, this will be paid in proportion to your course hours. If you complete your SKE course with compressed hours, this will affect your bursary amount. How and when your bursary is paid will vary by your training provider.
```

> If you train to teach in the 2022/23 academic year, you could be eligible for a bursary of £175 per week to complete your SKE course. If you do a part-time course and complete less than 25 hours a week, this will be paid in proportion to your course hours. If you complete your SKE course with compressed hours, this will affect your bursary amount. How and when your bursary is paid will vary by your training provider.

Reviewer 2: ```suggestion
If you train to teach in the 2022/23 academic year, you could be eligible for a bursary of £175 per week to complete your SKE course. If you do a part-time course and complete less than 25 hours a week, this will be paid in proportion to your course hours. If you complete your SKE course with compressed hours, this will also affect your bursary payment. How and when your bursary is paid will vary by your training provider.
```
Is `payment` more appropriate than `amount`? From what I'm understanding, it is the amount you get per week that is affected rather than the overall amount if you do compressed hours (but I might be wrong). Something to clarify with the SKE team.

Reviewer: would probably help to spell out to users exactly HOW it is affected

They're usually online, but some courses can be classroom based and some can be a mixture of both. 

Course lengths range from 8 to 28 weeks and are 25 hours per week. Your provider will decide which course length you need based on your gap in knowledge. Different providers may offer different course lengths.

You can do a part-time course and complete it over a longer period or complete it in a shorter time with compressed hours. Your SKE provider will be able to discuss your options with you.

## When to complete your course

You should complete your course before starting your teacher training. If you run out of time, your provider may allow you to continue your course at the same time as your teacher training.

## Get an SKE bursary

> If you train to teach in the 2023/24 academic year, you could be eligible for a bursary of £175 per week to complete your SKE course. If you do a part-time course and complete less than 25 hours a week, this will be paid in proportion to your course hours. If you complete your SKE course with compressed hours, this will affect your bursary amount. How and when your bursary is paid will vary by your training provider.

Reviewer: this is a very chunky para now, probably worth splitting up

Reviewer: can we clarify months rather than saying academic year? i.e. If you start your teacher training course between September 2023 and July 2024...

They're usually online, but some courses can be classroom based and some can be a mixture of both. 

Course lengths range from 8 to 28 weeks and are 25 hours per week. Your provider will decide which course length you need based on your gap in knowledge. Different providers may offer different course lengths.

You can do a part-time course and complete it over a longer period or complete it in a shorter time with compressed hours. Your SKE provider will be able to discuss your options with you.

## When to complete your course

You should complete your course before starting your teacher training. If you run out of time, your provider may allow you to continue your course at the same time as your teacher training.

## Get an SKE bursary

If you start your teacher training course between September 2023 and July 2024, you could be eligible for a bursary of £175 per week to complete your SKE course. 

If you do a part-time course and complete less than 25 hours a week, this will be paid in proportion to your course hours.

> If you complete your SKE course with compressed hours, this might reduce your bursary amount. How and when your bursary is paid will vary by your training provider.

Reviewer 2: ```suggestion
If you complete your SKE course with compressed hours, this might reduce your bursary amount. How and when your bursary is paid will vary by training provider.
```
